modules/setting/setting.go

```diff
@@ -1012,6 +1020,8 @@ func newService() {
 	Service.DefaultKeepEmailPrivate = sec.Key("DEFAULT_KEEP_EMAIL_PRIVATE").MustBool()
 	Service.DefaultAllowCreateOrganization = sec.Key("DEFAULT_ALLOW_CREATE_ORGANIZATION").MustBool(true)
 	Service.NoReplyAddress = sec.Key("NO_REPLY_ADDRESS").MustString("noreply.example.org")
+	Service.DefaultVisibility = sec.Key("DEFAULT_VISIBILITY").In("public", []string{"public", "limited", "private"})
```

Comment:
This list shouldn't be hard-coded but fetched from the hashmap above. If there's no go-func in stdlib to do that, use this:

```go
func ExtractKeysFromMapString(in map[string]interface{}) (keys []string) {
  for k, _ := range in {
    keys = append(keys, k)
  }
}
```